docs/docs/cli/download-sandbox-data.md

```diff
@@ -15,3 +15,91 @@ The script will ask you to choose a project, navigate the options with <kbd>↑<
 
 When the download process is complete, you visualise the results by following the printed instructions.
 For more details, see the [Open Encord Active](/cli/open-encord-active) page.
+
+
+If you have a dataset you'd like us to include in the sandbox please submit an issue on Github or contact us directly. We'd love to help!
+
+## Berkeley Deep Drive
+
+ **About the dataset**
+
+- **Research Paper:** BDD100K: A Diverse Driving Dataset for Heterogeneous Multitask Learning
+- **Authors:** Fisher Yu, Haofeng Chen, Xin Wang, Wenqi Xian, Yingying Chen, Fangchen Liu, Vashisht Madhavan, Trevor Darrell
+- **Dataset Size:** 1000 images & 12973 annotations
+- **Categories:** 8 classes
+- **License:** BSD 3-Clause License
+- **Release:** 21st September, 2020
+- **Read more:** [Webpage](https://bdd-data.berkeley.edu/) & [GitHub](https://github.com/bdd100k/bdd100ks)
+
+Sample pictures:
+![BDD dataset](../images/BDD.png)
+
+## COCO Validation 2017 Dataset
+
+ **About the dataset**
+
+- **Research Paper:** [Microsoft COCO: Common Objects in Context](https://arxiv.org/abs/1405.0312)
+- **Author:** [Tsung-Yi Lin](https://arxiv.org/search/cs?searchtype=author&query=Lin%2C+T) , [Michael Maire](https://arxiv.org/search/cs?searchtype=author&query=Maire%2C+M), [Serge Belongie](https://arxiv.org/search/cs?searchtype=author&query=Belongie%2C+S), [Lubomir Bourdev](https://arxiv.org/search/cs?searchtype=author&query=Bourdev%2C+L) , [Ross Girshick](https://arxiv.org/search/cs?searchtype=author&query=Girshick%2C+R), [James Hays](https://arxiv.org/search/cs?searchtype=author&query=Hays%2C+J), [Pietro Perona](https://arxiv.org/search/cs?searchtype=author&query=Perona%2C+P), [Deva Ramanan](https://arxiv.org/search/cs?searchtype=author&query=Ramanan%2C+D), [C. Lawrence Zitnic](https://arxiv.org/search/cs?searchtype=author&query=Zitnick%2C+C+L), [Piotr Dollár](https://arxiv.org/search/cs?searchtype=author&query=Doll%C3%A1r%2C+P)
+- **Dataset Size:**  5000 images, 4784 annotations
+- **Categories:**  7 plane categories
```

Comment:
81 classes no? Dont know what the 7 planes would be? Is that leftover from `rareplanes`?